mobile/src/libs/components/input/input.tsx

```diff
@@ -25,26 +26,44 @@ const Input = <T extends FieldValues>({
   errors,
   label,
   name,
+  secure,
   placeholder,
 }: Properties<T>): JSX.Element => {
+  const [isFocused, setIsFocused] = useState(false);
   const { field } = useFormController({ name, control });
 
   const { value, onChange, onBlur } = field;
 
   const error = errors[name]?.message;
   const hasError = Boolean(error);
 
+  const handleFocus = (): void => {
+    setIsFocused(true);
+  };
```

Comment:
Judging by B27 criteria, this should be a single-line arrow functions

Reply:
@OlesiaTr it was, but linter said to write it like this